### PR TITLE
Info panel string updating

### PR DESCRIPTION
**Pull request by sumitarora**
Updating properties of type `string` or `boolean` was not possible from info panel this fixes that issue
To test open InputOutput part of kitchen sink and try updating any of isOn, isDisabled, turn properties of InputOutput component

### CODE REVIEW

File: src/frontend/components/info-panel/info-panel.ts [hunk 1]
```diff
@@ -27,18 +27,20 @@ export class InfoPanel {
...
     private componentDataStore: ComponentDataStore,
     private userActions: UserActions,
+    private _ngZone: NgZone,
     @Inject(ElementRef) private elementRef: ElementRef
   ) {
 
     this.componentDataStore.dataStream
-      .filter((data: any) => data.action &&
-        data.action !== UserActionType.GET_DEPENDENCIES)
-      .debounce((x) => {
```
Comment (vanessayuenn) [inline src/frontend/components/info-panel/info-panel.ts]: do we not need the `debounce` here anymore for performance sake?

Comment (sumitarora) [inline src/frontend/components/info-panel/info-panel.ts]: No, we don't we fixed the batch call issue for which it was added in mutation observer one forgot to  remove this.

Comment (vanessayuenn) [inline src/frontend/components/info-panel/info-panel.ts]: :+1:

**Comment by vanessayuenn** [comment]
LGTM!

**Pull request merged**